src/items/controllers/item.controller.ts

```diff
@@ -25,6 +25,14 @@ export class ItemsController {
     return this.itemService.create(itemRequestDto);
   }
 
+  @Patch('/:itemId/buy')
+  buyItem(
+    @Param('itemId') itemId: string,
+    @Body() { address }: { address: string }
```

Comment:
Please add validation to the address field. It should not be empty and should be ethereum address